packages/thirdweb/src/extensions/modular/ModularCoreUpgradeable/write/installPublishedExtension.ts

```diff
@@ -0,0 +1,53 @@
+import type { ThirdwebContract } from "../../../../contract/contract.js";
+import type { Account } from "../../../../wallets/interfaces/wallet.js";
+import { getOrDeployInfraForPublishedContract } from "../../../../contract/deployment/utils/bootstrap.js";
+import type { ThirdwebClient } from "../../../../client/client.js";
+import type { Chain } from "../../../../chains/types.js";
+import { installExtension } from "../../__generated__/ModularCoreUpgradeable/write/installExtension.js";
+
+export type InstallPublishedExtensionOptions = {
+  client: ThirdwebClient;
+  chain: Chain;
+  account: Account;
+  contract: ThirdwebContract;
+  extensionName: string;
+  publisherAddress: string;
```

Comment:
make this optional and default to THIRDWEB_DEPLOYER? here and in the other extensions